src/Signal-Keithley_4200-SCS/main.py

```diff
@@ -198,7 +197,12 @@ def connect(self):
                 # not supported yet with pylptlib -> # self.inst = inst
                 self.param = param
 
-            ret = self.lpt.initialize()
+            try:
+                ret = self.lpt.initialize()
+            except ConnectionRefusedError as e:
+                debug("Unable to connect to a lptlib server application running on the 4200-SCS. Please check your"
+                      "network settings and make sure the server application is running.")
```

Comment:
```suggestion
                debug("Unable to connect to a lptlib server application running on the 4200-SCS. Please check your "
                      "network settings and make sure the server application is running.")
```
space required, I think